### PR TITLE
[ISSUE #73] Support additional volume mount for console deployment

### PR DESCRIPTION
**Pull request by drivebyer**
## What is the purpose of the change
support https://github.com/apache/rocketmq-operator/issues/73

## Verifying this change

截屏2023-09-18 16 23 16
We can login with custom user/pass

**Please go through this checklist to help us incorporate your contribution quickly and easily.**

Notice: `It would be helpful if you could finish the following checklist (the last one is not necessary) before request the community to review your PR`.

- [ ] Make sure there is a [Github issue](https://github.com/apache/rocketmq-operator/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Check RBAC rights for Kubernetes roles.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. 
- [ ] Run `make docker-build` to build docker image for operator, try your changes from Pod inside your Kubernetes cluster, **not just locally**. Also provide screenshots to show that the RocketMQ cluster is healthy after the changes. 
- [ ] Before committing your changes, remember to run `make manifests` to make sure the CRD files are updated. 
- [ ] Update documentation if necessary.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Comment by drivebyer** [comment]
PTAL @caigy

File: example/rocketmq_v1alpha1_rocketmq_cluster.yaml [hunk 1]
```diff
@@ -163,6 +163,28 @@ spec:
         spec:
           containers:
             - name: console
+              env:
+                - name: ROCKETMQ_CONFIG_LOGINREQUIRED
+                  value: "true"
```
Comment (caigy) [inline example/rocketmq_v1alpha1_rocketmq_cluster.yaml:168]: If login is required by default, you'd better also update the document.

File: example/rocketmq_v1alpha1_rocketmq_cluster.yaml [hunk 2]
```diff
@@ -163,6 +163,28 @@ spec:
...
               image: apacherocketmq/rocketmq-console:2.0.0
               ports:
-                - containerPort: 8080
\ No newline at end of file
+                - containerPort: 8080
+              volumeMounts:
+                - name: console-config
+                  mountPath: /tmp/rocketmq-console/data
```
Comment (caigy) [inline example/rocketmq_v1alpha1_rocketmq_cluster.yaml:174]: It seems a bit curious to place the configuration files under `/tmp`.

**Review by caigy — CHANGES_REQUESTED** [review]

File: example/rocketmq_v1alpha1_rocketmq_cluster.yaml (hunk 1 above)
Comment (drivebyer) [reply example/rocketmq_v1alpha1_rocketmq_cluster.yaml:168]: Login is not need required by default, it should be just a guide for user, i would fix it.

File: example/rocketmq_v1alpha1_rocketmq_cluster.yaml (hunk 2 above)
Comment (drivebyer) [reply example/rocketmq_v1alpha1_rocketmq_cluster.yaml:176]: It's default directory of console data, see https://github.com/apache/rocketmq-dashboard/blob/6456630324e3ff22217a6da221a64c09bea82a72/src/main/resources/application.yml#L51

**Comment by drivebyer** [comment]
see https://github.com/apache/rocketmq-operator/pull/182

**Pull request closed without merging**